src/data/status-effect.ts

```diff
@@ -32,20 +33,20 @@ export class Status {
 }
 
 export function getStatusEffectObtainText(statusEffect: StatusEffect, sourceText?: string): string {
-  const sourceClause = sourceText ? ` ${statusEffect !== StatusEffect.SLEEP ? "by" : "from"} ${sourceText}` : "";
+  const sourceClause = sourceText ? ` ${statusEffect !== StatusEffect.SLEEP ? i18next.t("statusEffectSourceClause:by") : i18next.t("statusEffectSourceClause:from")} ${sourceText}` : "";
   switch (statusEffect) {
   case StatusEffect.POISON:
-    return `\nwas poisoned${sourceClause}!`;
+    return i18next.t("statusEffectObtainText:poison", { sourceClause: sourceClause });
   case StatusEffect.TOXIC:
-    return `\nwas badly poisoned${sourceClause}!`;
+    return i18next.t("statusEffectObtainText:toxic", { sourceClause: sourceClause });
   case StatusEffect.PARALYSIS:
-    return ` was paralyzed${sourceClause}!\nIt may be unable to move!`;
+    return i18next.t("statusEffectObtainText:paralysis", { sourceClause: sourceClause });
   case StatusEffect.SLEEP:
-    return `\nfell asleep${sourceClause}!`;
+    return i18next.t("statusEffectObtainText:sleep", { sourceClause: sourceClause });
   case StatusEffect.FREEZE:
-    return `\nwas frozen solid${sourceClause}!`;
+    return i18next.t("statusEffectObtainText:freeze", { sourceClause: sourceClause });
   case StatusEffect.BURN:
-    return `\nwas burned${sourceClause}!`;
+    return i18next.t("statusEffectObtainText:burn", { sourceClause: sourceClause });
```

Comment:
```suggestion
    return i18next.t("statusEffectObtainText:burn", { sourceClause });
```

[sauce](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#property_definitions)